buildenv/jenkins/openjdk8_x86-64_linux_docker

```diff
@@ -3,7 +3,7 @@
 	Set parameter TARGET(openjdk, system, external, perf, jck etc.).
 	Set UPSTREAM_JOB_NAME(version_build_arch_os, for example: openjdk8_build_x86-64_linux)
 	Set JVM_VERSION(openjdk8, openjdk8-openj9, openjdk9, openjdk8-openj9, etc.*/
-node('linux&&x64&&test&&sw.tool.docker') {
+node('test-packet-ubuntu1604-x64-1&&test-packet-ubuntu1604-x64-3&&test-scaleway-ubuntu1604-x64-1') {
```

Comment:
Is this intended? We should not hardcode machine name.  And I do not think this will match any machine.

Reply:
Please ignore this change. I only did it for testing purposes to avoid a failing machine.

Reply:
Could you remove it from PR? If we merge the PR, this change will be in.

Reply:
@llxia - of course. The idea is to remove it before merging - which is what I have done.